packages/pancake-uikit/src/components/Slider/Slider.tsx

```diff
@@ -1,12 +1,16 @@
 import React, { ChangeEvent } from "react";
 import { Box } from "../Box";
-import { BunnySlider, BarBackground, BarProgress, BunnyButt, StyledInput, SliderLabel } from "./styles";
+import {
+  BunnySlider,
+  BarBackground,
+  BarProgress,
+  BunnyButt,
+  StyledInput,
+  SliderLabel,
+  SliderLabelContainer,
+} from "./styles";
 import SliderProps from "./types";
 
-// We need to adjust the offset as the percentage increases, as 100% really is 100% - label width.
-// The number 10 is arbitrary, but seems to work...
-const LABEL_OFFSET = 10;
-
```

Comment:
Can you explain why this was necessary to remove?

Reply:
This is not used in the current change.